addons/mass_mailing/data/mass_mailing_demo.xml

```diff
@@ -58,7 +58,7 @@
 
         <record id="mass_mail_1" model="mailing.mailing">
             <field name="name">Newsletter 1</field>
-            <field name="subject" eval="'%s Newsletter' % (DateTime.today().strftime('%B'))"/>
+            <field name="subject">Monthly Newsletter</field>
```

Comment:
@tde-banana-odoo I was wondering why the translation was changing from "October Newsletter" to "November Newsletter"